### DISCUSSION TITLE
Filtering via a relational field with multiple collections doesn't work

### DISCUSSION THREAD

**Discussion by jmurth1234** [payloadcms/payload#2837] [Feature Requests & Ideas]
### Link to reproduction

https://github.com/rymate1234/payload/tree/reprod/multi-relational-filter

### To Reproduce

1. Load CMS
2. Upload a media or media alt to have one to test
3. try to filter the posts by it
4. get the error:

[23:09:23] ERROR (payload): QueryError: The following path cannot be queried: associatedMedia
    at Function.buildQuery (/Users/murthick/Projects/Mythical/payload/src/mongoose/buildQuery.ts:591:15)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async find (/Users/murthick/Projects/Mythical/payload/src/collections/operations/find.ts:103:17)
    at async findHandler (/Users/murthick/Projects/Mythical/payload/src/collections/requestHandlers/find.ts:23:20)

> 

### Describe the Bug

Filtering by multiple relations doesn't work, presumably because the filter should be `associatedMedia__value` 

### Payload Version

1.7.5, but reproduced in latest master

**Comment by PatrikKozak** [comment]
Hey @rymate1234, thanks for the input and reaching out to us with your findings! I went ahead and reviewed the issue you highlighted and this actually looks to be more of an enhancement rather than a bug.

There is some complexity and potential impact on performance with this kind of feature. I am going to convert this into a discussion for a thorough evaluation. I'll tag it as a potential new feature, and encourage you to share any further insights in the discussion. 

However, as a workaround for now, you could query your relationships directly instead of through the relationship field and then those IDs to return your documents.

Does that make sense?

Thanks again and let me know!

**Reply by jmurth1234** [reply]
How would I implement this in the CMS UI? We have users who want to filter a collection via the list view filters, and that's where we're specifically running into this issue

![image](https://github.com/payloadcms/payload/assets/581104/63dbacad-879a-4e08-9802-22dd7de03831)

I had posted about this on the Discord https://discord.com/channels/967097582721572934/1118500809450209330/1118500809450209330
